docs/frameworks/sveltekit.md

```diff
@@ -4,212 +4,214 @@ title: SvelteKit | Frameworks
 
 # SvelteKit
 
+::: warning
+From SvelteKit version `1.0.0-next.358+`, SvelteKit is just another Vite plugin, and latest versions will also require you to update your application to use Vite 3.
+:::
+
 ::: info
 For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Svelte](/frameworks/svelte) entry.
 :::
 
+::: tip
 You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) to disable it on your application.
+:::
+
+## SvelteKit Vite Plugin
+
+`vite-plugin-pwa` provides the new `ViteSvelteKitPWA` plugin that will allow you to use `vite-plugin-pwa` in your SvelteKit applications.
 
-Since `SvelteKit` uses `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA` will only register the service worker on build, it is aligned with the current behavior of [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker).
+To update your project to use the new `vite-plugin-pwa` for SvelteKit, you only need to change the Vite config file (you don't need oldest `pwa` and `pwa-configuration` modules):
```

Comment:
It'd probably make sense to write this aimed primarily towards people setting up new projects rather than those upgrading from older versions
```suggestion
To update your project to use the new `vite-plugin-pwa` for SvelteKit, you only need to add the plugin in the Vite config file (if you're upgrading from a prior version, you no longer need the `pwa` and `pwa-configuration` modules):
```